fenix/app/metrics.yaml

```diff
@@ -4075,6 +4075,40 @@ settings:
       tags:
         - Sync
         - Settings
+  passwords:
+    type: event
+    description: |
+      User has tapped Passwords in settings menu.
+    bugs:
+      - https://bugzilla.mozilla.org/show_bug.cgi?id=1841152
+    data_reviews:
+      - https://bugzilla.mozilla.org/show_bug.cgi?id=1841152
```

Comment:
data review link should be https://github.com/mozilla-mobile/firefox-android/pull/2802.  Same with the other ones below.

Reply:
D'oh! Fixed